cv_next/config/contents.ts

```diff
@@ -5,3 +5,9 @@ export const heroHeader: HeroHeader = {
   subheader: `Easy to setup. Customizable. Quick. Responsive.`,
   image: `/hero-img.webp`,
 }
+
+export const inactiveHeader: HeroHeader = {
```

Comment:
Ok so this file is a demo file, I don't think this style is good. This header details are not "configuration", they are constants. Please move this (and if you want the one above as well) to a `constants` folder (in a file of name e.g. `"headerDetails"` or something like that)